### DISCUSSION TITLE
[Unstable] Odd Riot Shield behaviour when dragging into hotbar

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14540] [Bug reports]
### What happened?

Trying to drop a riot shield into the hotbar causes strange behavior where it's simultaneous in the bag slot but also being held.

https://github.com/user-attachments/assets/2c5b9bd2-f98d-44fb-b210-4a507f517f5c

### Reproduction steps

1. Get a riot shield
2. Try to drop it into your hotbar

### Version

v1.6.3.0 (unstable)   **<-- The bug report form still has v1.6.101.0 (unstable) as the option, instead of the current v1.6.3.0**

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/dfaa4e51b9fcbd43f99ca1969341e8509994cb18, fix coming up in the next unstable build.
